### ISSUE TITLE
[BUG]: Networks are not persistant if docker application is stopped

### ISSUE THREAD

**Issue by Serph91P** [azukaar/Cosmos-Server#98]
### What happened?

Created a docker from market aswell as from a compose file. Had to stop them because some files where missing and debugging. After I wanted to start the applications again the reported that they cannot start because the network is missing.

### What should have happened?

Applications start because network is not gone.

### How to reproduce the bug?

1. Fresh install of cosmos-server
2. Install a compose file or from the market
3. Test if it is working.
4. Stop application
5. After some time start application again.
6. Cannot be startet

### Relevant log output

```shell
2023-09-26 17:00:43
 2023/09/26 17:00:43 [ERROR] ManageContainer: restart : Error response from daemon: Cannot restart container a7bd81a33470d059ebf29defe52a937d4ef9814c51263487d1710e6ca65a6d9f: network 6bb0d1af24f3f731f54d3a314f0d1890b193a2137049c0b226b6d451fe048593 not found
2023-09-26 17:00:43
 2023/09/26 17:00:43 [ERROR] HTTP Request returned Error 500 : Internal server error: Error response from daemon: Cannot restart container a7bd81a33470d059ebf29defe52a937d4ef9814c51263487d1710e6ca65a6d9f: network 6bb0d1af24f3f731f54d3a314f0d1890b193a2137049c0b226b6d451fe048593 not found :
```

### Other details

I did not delete the networks. Now I cannot start any of them all have no networks.

### System details

- OS: Linux Ubuntu-2204-jammy-amd64-base 5.15.0-79-generic #86-Ubuntu SMP Mon Jul 10 16:07:21 UTC 2023 x86_64 x86_64 x86_64 GNU/Linux
- Browser: Firefox
- Version: 117.0.1

**Comment by azukaar** [comment]
I'm guessing docker report the network has being empty if the containers are stopped, so Cosmos clean them out, need to look into a solution

**Comment by azukaar** [comment]
fixed in 0.15
